.github/workflows/dotnet.yml

```
@@ -0,0 +1,29 @@
+name: .NET CI
+
+on:
+  push:
+    branches: [ "lab_1" ]
+  pull_request:
+    branches: [ "lab_1" ]
```

Я бы сразу акшон завязывал на разные ветки.
```suggestion
  push:
    branches: [ "*" ]
  pull_request:
    branches: [ "main" ]
```
У вас акшон не запустился.